plugins/command-manager/src/main/java/com/wazuh/commandmanager/settings/PluginSettings.java

```diff
@@ -82,6 +82,7 @@ private PluginSettings(@NonNull final Settings settings) {
         this.jobSchedule = JOB_SCHEDULE.get(settings);
         this.jobPageSize = JOB_PAGE_SIZE.get(settings);
         this.jobKeepAlive = JOB_KEEP_ALIVE.get(settings);
+        log.error("Settings loaded: {}", this.toString());
```

Comment:
May be better if the log level is `info` or `debug`.

Reply:
Looking at the [issue](https://github.com/wazuh/wazuh-indexer-plugins/issues/225), maybe `debug` is the correct level for this log

Reply:
Fixed in 66074be5f391541041a592ba2007a6ed09f39abd